classes/fsl-kernel-localversion.bbclass

```diff
@@ -20,7 +20,7 @@ do_kernel_localversion() {
 
 	# Fallback for recipes not able to use LINUX_VERSION_EXTENSION
 	if [ "${@bb.data.inherits_class('kernel-yocto', d)}" = "False" ]; then
-		echo "CONFIG_LOCALVERSION=${LOCALVERSION}" >> ${B}/.config
+		echo "CONFIG_LOCALVERSION=\"${LOCALVERSION}\"" >> ${B}/.config
```

Comment:
Avoiding escape chars is more readable in my opinion
```
echo '"CONFIG_LOCALVERSION="${LOCALVERSION}"' >> ${B}/.config
```

Reply:
Avoid the escales, it is nicer indeed.

Reply:
@otavio updated the commit using single quotes to avoid escaping the double quotes.